activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -18,6 +20,16 @@ module FinderMethods
     #   Person.find([1])        # returns an array for the object with ID = 1
     #   Person.where("administrator = 1").order("created_on DESC").find(1)
     #
+    # ==== Find a composite primary key model
+    #   TravelRoute.primary_key = [:origin, :destination]
+    #   TravelRoute.find("Ottawa", "London") # returns the object for the route from Ottawa to London
```

Comment:
Does `find(k1, k2)` (no Array) work for CPK models? 🤔 

```ruby
require "active_record"
require "minitest/autorun"
require "logger"

ActiveRecord::Base.establish_connection(adapter: "sqlite3", database: ":memory:")
ActiveRecord::Base.logger = Logger.new(STDOUT)

ActiveRecord::Schema.define do
  create_table :travel_routes, primary_key: false, force: true do |t|
    t.string :origin
    t.string :destination
  end
end

class TravelRoute < ActiveRecord::Base
  self.primary_key = [:origin, :destination]
end

class CpkTest < Minitest::Test
  def test_cpk
    TravelRoute.create!(origin: "Ottawa", destination: "London")

    TravelRoute.find("Ottawa", "London")
  end
end
```
```
Error:
CpkTest#test_cpk:
TypeError: wrong argument type String (must respond to :each)
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/relation/finder_methods.rb:540:in `zip'
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/relation/finder_methods.rb:540:in `block in find_some_ordered'
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/relation/finder_methods.rb:540:in `map'
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/relation/finder_methods.rb:540:in `find_some_ordered'
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/relation/finder_methods.rb:505:in `find_some'
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/relation/finder_methods.rb:479:in `find_with_ids'
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/relation/finder_methods.rb:69:in `find'
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/querying.rb:23:in `find'
    /Users/adriannachang/src/github.com/rails/activerecord/lib/active_record/core.rb:244:in `find'
    cpk_find_test.rb:24:in `test_cpk'
```

Reply:
>Does find(k1, k2) (no Array) 

Actually it doesn't work and wasn't supposed to 🙈 
Thanks! Will fix it